src/store.js

```diff
@@ -1,10 +1,85 @@
-import Vue from 'vue'
-import Vuex from 'vuex'
+import Vue from "vue";
```

Comment:
we should take a look at your linter, inside javascript it should be single quoted